website/src/pages/index.js

```diff
@@ -36,7 +36,11 @@ const HomepageHeader = () => {
             Watch the tutorials
           </Link>
           <Link
-            className={clsx('button button--outline button--primary button--lg', styles.getStarted)}
+            className={clsx(
+              'button button--outline button--primary button--lg demo',
```

Comment:
where is this class `demo` applied?

Reply:
in the demo button, in the landing page itself.